CHANGELOG.md

```diff
@@ -45,31 +101,71 @@
     * Added a Preferences settings to preserve line width while resizing a
       selection (#2011)
     * Added a Preferences setting to change the snap grid size (#1920).
-    * Fixed a bug in the object selection algorithm (#2478)
-* Pen
+    * Added Preference settings to configure the grid size (#1920).
```

Comment:
Lines 103 and 104 both reference issue #1920... This seems a bit confusing.

Thank you for working on this!

Reply:
Thanks for the catch.